codespell_lib/data/dictionary.txt

```diff
@@ -58421,6 +58524,7 @@ thefore->therefore
 thei->their, they,
 theif->thief
 theifs->thieves
+theim->Thiem, them,
```

Comment:
For the sake of completeness, although probably rare:
```suggestion
theim->Thiem, them, theism,
```
Who's Thiem by the way?

Reply:
It could be [a few well known people](https://en.wikipedia.org/wiki/Thiem), and my use case is [an equation](https://en.wikipedia.org/wiki/Aquifer_test#Steady-state_Thiem_solution). This specific entry helped me a few days ago [here](https://github.com/MODFLOW-USGS/modflow6/commit/4c14af9beca193142adb49d59178ad470426a222).